thirdeye-ui/src/app/locale/languages/en-us.json

```diff
@@ -714,6 +715,7 @@
         "property-value-placeholder": "Type and press enter to add values",
         "provide-an-optional-description-for-this-alert": "Provide an optional description for this alert",
         "query-example": "e.g. country='IN' AND device = ‘iPhone’",
+        "query-timeout-error": "Raise a ticket with support team, to fix query performance issues",
```

Comment:
I think it should be prefixed with `pinot-`. Also it'll make it easier to add more PINOT error code and pinot error messages in the same place.  
We also have MySQL queries.